tests/skipped_tests.tbl

```diff
@@ -557,12 +557,21 @@ tests/third_party/cupy/math_tests/test_sumprod.py::TestNansumNanprodLong_param_1
 tests/third_party/cupy/math_tests/test_sumprod.py::TestNansumNanprodLong_param_15_{axis=0, func='nanprod', keepdims=False, shape=(20, 30, 40), transpose_axes=False}::test_nansum_axis_transposed
 tests/third_party/cupy/math_tests/test_sumprod.py::TestNansumNanprodLong_param_9_{axis=0, func='nanprod', keepdims=True, shape=(2, 3, 4), transpose_axes=False}::test_nansum_all
 tests/third_party/cupy/math_tests/test_sumprod.py::TestNansumNanprodLong_param_9_{axis=0, func='nanprod', keepdims=True, shape=(2, 3, 4), transpose_axes=False}::test_nansum_axis_transposed
+tests/third_party/cupy/math_tests/test_sumprod.py::TestSumprod::test_external_sum_axis
 tests/third_party/cupy/math_tests/test_sumprod.py::TestSumprod::test_prod_all
 tests/third_party/cupy/math_tests/test_sumprod.py::TestSumprod::test_prod_axis
 tests/third_party/cupy/math_tests/test_sumprod.py::TestSumprod::test_prod_dtype
 tests/third_party/cupy/math_tests/test_sumprod.py::TestSumprod::test_sum_all2
 tests/third_party/cupy/math_tests/test_sumprod.py::TestSumprod::test_sum_all_transposed2
+tests/third_party/cupy/math_tests/test_sumprod.py::TestSumprod::test_sum_axes
+tests/third_party/cupy/math_tests/test_sumprod.py::TestSumprod::test_sum_axes2
```

Comment:
These tests stops working in this PR? It is not expected.

Reply:
I already explained to @Rubtsowa why it happens. Previously axis parameter was in kwargs. There is a condition
https://github.com/IntelPython/dpnp/blob/e3d18674e64ba2c65c47fea0e3c4d0346898d532/dpnp/dpnp_iface_mathematical.py#L640-L643
so when axis parameter was defined we had a fallback to original Numpy. That is why previously tests where passed.